### ISSUE TITLE
Name

### ISSUE THREAD

**Issue by llwyd** [llwyd/wavPlay#3]
Does anyone have any good suggestions for a name?  "wavPlay" is a bit... obvious.  I definitely think we could come up with something cooler.

**Comment by xiewen112** [comment]
Extracting human voice maybe a good idea. We add a function to extract human voice from noisy surrounding or songs. That's must be cool and useful in many areas.

**Comment by llwyd** [comment]
I was thinking about Pokemon on the bus this morning and the idea of the name "thunderWave" came to my head.

**Comment by xiewen112** [comment]
It's a good name. 

    On Wednesday, March 9, 2016 11:01 AM, Tom <notifications@github.com> wrote:
 

 I was thinking about Pokemon on the bus this morning and the idea of the name "thunderWave" came to my head.—

**Comment by llwyd** [comment]
So we are leaning towards the name thunderWave, any objections?

**Comment by xiewen112** [comment]
No, I agree with it On 2016-03-19 12:53 , Tom Wrote: So we are leaning towards the name thunderWave, any objections? — You are receiving this because you commented. Reply to this email directly or view it on GitHub

**Comment by llwyd** [comment]
Project is now renamed

**Comment by xiewen112** [comment]
Pretty cool! On 2016-03-24 14:37 , Tom Wrote: Project is now renamed — You are receiving this because you commented. Reply to this email directly or view it on GitHub
